spec/std/openssl/pkcs5_spec.cr

```diff
@@ -12,4 +12,15 @@ describe OpenSSL::PKCS5 do
       OpenSSL::PKCS5.pbkdf2_hmac_sha1("password", "salt", iterations, key_size).hexstring.should eq expected
     end
   end
+
+  it "computes pbkdf2_hmac" do
+    [
```

Comment:
You could use a tuple here instead of an array

Reply:
I will try because I just copied from original test case